draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1123,37 +1123,29 @@ the media type is encoded as an SD-JWT.
 
 All of the following examples are non-normative.
 
-## Example 2a: Handling Structured Claims {#example-simple_structured}
+## Example 2: Handling Structured Claims {#example-simple_structured}
 
 This example uses the following object as the set of claims that the Issuer is issuing:
 
 <{{examples/simple_structured/user_claims.json}}
 
-Note that in contrast to Example 1, here the Issuer decided to create a structured object for the `address` claim, allowing for separate disclosure of the individual members of the claim.
+In contrast to Example 1, here the Issuer decided to create a structured object for the `address` claim, allowing for separate disclosure of the individual members of the claim, and also added decoy digests to prevent the Verifier from deducing the true number of claims in the address object. The following payload is used for the SD-JWT:
```

Comment:
The example has decoys in the address and at the top level too. So this text should probably be more general about the decoys.  Maybe just:
```suggestion
In contrast to Example 1, here the Issuer decided to create a structured object for the `address` claim, allowing for separate disclosure of the individual members of the claim, and also added decoy digests to prevent the Verifier from deducing the true number of claims. The following payload is used for the SD-JWT:
```